src/pyroboplan/models/panda.py

```diff
@@ -4,6 +4,8 @@
 import numpy as np
 import os
 import pinocchio
+import hppfcl as fcl
```

Comment:
This import still works for compatibility, but you should be using the new name `coal` -- see the other examples